.travis.yml

```diff
@@ -258,24 +270,30 @@ matrix:
         - ./build-support/bin/travis-ci.sh -ez
 
     # Rust Clippy on Linux
-    - name: "Rust Clippy on Linux"
+    - << *runs_on_ci
+      << *with_fuse
+      name: "Rust Clippy on Linux"
       os: linux
       dist: trusty
       sudo: required
       stage: Test Pants
       language: python
       python: *python_version
-      before_install:
-        - sudo apt-get install -y pkg-config fuse libfuse-dev
-        - sudo modprobe fuse
-        - sudo chmod 666 /dev/fuse
-        - sudo chown root:$USER /etc/fuse.conf
       before_script:
         - ulimit -c unlimited
         - ulimit -n 8192
       script:
         - ./build-support/bin/travis-ci.sh -bs
 
+    #- if: type == "cron"
```

Comment:
Whoa... is this syntax? A comment above this explaining that the next line is a magic comment would be good.

Reply:
I think this is just a comment. If so, maybe xx it?